transform/models/staging/_stg_models.yml

```diff
@@ -7,98 +7,98 @@ models:
     columns:
       - name: EVENT_DATE
         description: The date when the event was logged
-        data_tests:
-          - data_gaps:
-              start_date: "{{ var('ga_data_start_date') }}"
-              end_date: "current_date - 2"
+        # data_tests:
+        #   - data_gaps:
+        #       start_date: "{{ var('ga_data_start_date') }}"
+        #       end_date: "current_date - 2"
       - name: EVENT_TIMESTAMP
         description: The time (in microseconds, UTC) when the event was logged
       - name: EVENT_NAME
         description: The name of the event
       - name: EVENT_PARAMS
         description: Event parameter keys and values associated with the event
-      - name: EVENT_PREVIOUS_TIMESTAMP
-        description: >
-          The time (in microseconds, UTC) when the event was previously logged.
-      - name: EVENT_VALUE_IN_USD
-        description: The event's "value" parameter (converted to USD)
-      - name: EVENT_BUNDLE_SEQUENCE_ID
-        description: The sequential ID of the bundle in which these events were uploaded
-      - name: EVENT_SERVER_TIMESTAMP_OFFSET
-        description: >
-          Timestamp offset between collection time and upload time (in micros)
-      - name: USER_ID
-        description: ID assigned to a user, has to be set by GA site code
+      # - name: EVENT_PREVIOUS_TIMESTAMP
+      #   description: >
+      #     The time (in microseconds, UTC) when the event was previously logged.
+      # - name: EVENT_VALUE_IN_USD
+      #   description: The event's "value" parameter (converted to USD)
+      # - name: EVENT_BUNDLE_SEQUENCE_ID
+      #   description: The sequential ID of the bundle in which these events were uploaded
+      # - name: EVENT_SERVER_TIMESTAMP_OFFSET
+      #   description: >
+      #     Timestamp offset between collection time and upload time (in micros)
+      # - name: USER_ID
+      #   description: ID assigned to a user, has to be set by GA site code
```

Comment:
If these columns no longer exist in the model, let's remove the documentation all-together.